src/mvTables.cpp

```diff
@@ -72,7 +72,9 @@ void mvTableColumn::handleSpecificKeywordArgs(PyObject* dict)
 	flagop("prefer_sort_descending", ImGuiTableColumnFlags_PreferSortDescending, _flags);
 	flagop("indent_enable", ImGuiTableColumnFlags_IndentEnable, _flags);
 	flagop("indent_disable", ImGuiTableColumnFlags_IndentDisable, _flags);
-
+	flagop("angle_header", ImGuiTableColumnFlags_AngledHeader, _flags);
+	flagop("disabled", ImGuiTableColumnFlags_Disabled, _flags);
+	flagop("no_header_label", ImGuiTableColumnFlags_NoHeaderLabel, _flags); 
```

Comment:
:exclamation::exclamation: This one doesn't work. The gotcha is that ImGui handles this option in `ImGui::TableHeadersRow()`, but DPG doesn't call that function - instead, it implements its own header row based on `ImGuiTableRowFlags_Headers` and `ImGui::TableHeader()`.

Not sure if we really need this option, but if we do, we probably have to copy-paste more stuff from `ImGui::TableHeadersRow()` (and I somewhat dislike the idea). As far as I can see, the only reason for having a custom header in DPG is that we can bind a tooltip to a table column, and the tooltips need to be interspersed with `TableHeader` calls (they show up when the header is hovered).

Reply:
I should have fixed both this and the `angled_header`.. I honestly don't know if I chose the best way, but you'll tell me

Reply:
I've looked at the updated code, and there's a problem in there.

If table columns have tooltips attached to them, and angled headers are enabled, the tooltips will actually monitor hovered state of a wrong item - of the one rendered right before the table. Here's what it looks like:

![angled-header-tooltips1](https://github.com/hoffstadt/DearPyGui/assets/104593001/71654927-8b07-43e2-9982-549d0e001162)

Here's a fix for this:

https://github.com/v-ein/DearPyGui-fixes/compare/porting...bugfix/table-header-tooltips

Should look like this:

![angled-header-tooltips2](https://github.com/hoffstadt/DearPyGui/assets/104593001/e8f50bf9-4b47-4912-b82f-d01ad8b1e13c)

The script I used to test it:

```py
import dearpygui.dearpygui as dpg

dpg.create_context()
dpg.create_viewport(title="Test", width=700, height=750)

with dpg.window(height=500) as wnd:
    dpg.add_button(label="Don't click me, I'm a placeholder")
    with dpg.table(
        hideable=True,
        borders_innerH=True, borders_innerV=True, borders_outerH=True, borders_outerV=True,
        policy=dpg.mvTable_SizingFixedFit) as table:

        dpg.add_table_column(label="Col 1", tag="both-col", width_fixed=True)
        with dpg.tooltip(dpg.last_item()):
            dpg.add_text("I'm a tooltip for column 1")
        dpg.add_table_column(label="Col 2", tag="angled-col", width_stretch=True)
        with dpg.tooltip(dpg.last_item()):
            dpg.add_text("I'm a tooltip for column 2")
        dpg.add_table_column(label="Col 3", tag="no-label-col", width_fixed=True)
        with dpg.tooltip(dpg.last_item()):
            dpg.add_text("I'm a tooltip for column 3")
        dpg.add_table_column(label="Col 4", width_fixed=True)
        with dpg.tooltip(dpg.last_item()):
            dpg.add_text("I'm a tooltip for column 4")
        with dpg.table_row():
            dpg.add_button(label="Lorem")
            dpg.add_button(label="ipsum")
            dpg.add_button(label="dolor")

    dpg.add_checkbox(label="Angled header", callback=lambda s, checked: (dpg.configure_item("angled-col", angled_header=checked),  dpg.configure_item("both-col", angled_header=checked)))
    dpg.add_checkbox(label="No label", callback=lambda s, checked: (dpg.configure_item("no-label-col", no_header_label=checked),  dpg.configure_item("both-col", no_header_label=checked)))

dpg.setup_dearpygui()
dpg.show_viewport()
dpg.start_dearpygui()
dpg.destroy_context() 
```